resources/voting-controls.js

```diff
@@ -0,0 +1,149 @@
+let voted_points = null;
+let voted_note = null;
+let voted_date = null;
+let has_voted = false;
+
+function set_default_vote_data(vote_points, vote_date) {
+    has_voted = true;
+    voted_points = vote_points;
+    voted_date = vote_date;
+    voted_note = JSON.parse(document.getElementById('voted_note').textContent);
+}
+
+let vote_url = '';
+let delete_vote_url = '';
+
+function set_voting_urls(new_vote_url, new_delete_vote_url) {
+    vote_url = new_vote_url;
+    delete_vote_url = new_delete_vote_url;
+}
+
+let translator = null;
+function set_translated_strings(translated_strings){
```

Comment:
```suggestion
function set_translated_strings(translated_strings) {
```
Please update other functions as well.